aws/resource_aws_security_group_test.go

```diff
@@ -642,7 +642,7 @@ func TestAccAWSSecurityGroup_IPRangeAndSecurityGroupWithSameRules(t *testing.T)
 			{
 				ResourceName:            resourceName,
 				ImportState:             true,
-				ImportStateCheck:        testAccAWSSecurityGroupImportStateCheckFunc(4),
+				ImportStateCheck:        testAccAWSSecurityGroupImportStateCheckFunc(1),
```

Comment:
Drive-by note: For the testing updates, we can remove the `ImportStateCheck` and should ensure the `ImportStateVerify`/`ImportStateVerifyIgnore` are uncommented. 👍

Reply:
@bflad  Modified the acceptance tests to remove `ImportStateCheck` and uncommented `ImportStateVerify`/`ImportStateVerifyIgnore`.
Well, in all except 2 cases related to the drift rules. 
In rules with cidr_block drift, import only creates a single ingress rule with the cidr_blocks de-normalized. And during subsequent apply, its normalized to create the 2 ingress rules. So those checks don’t lend themselves to these cases.